android/src/main/kotlin/com/sidlatau/flutteremailsender/FlutterEmailSenderPlugin.kt

```diff
@@ -93,18 +93,16 @@ class FlutterEmailSenderPlugin(private val registrar: Registrar)
                 intent.putExtra(Intent.EXTRA_BCC, listArrayToArray(bcc))
             }
         }
-
-        if (options.hasArgument(ATTACHMENT_PATH)) {
-            val attachmentPath = options.argument<String>(ATTACHMENT_PATH)
-            if (attachmentPath != null) {
+        if (options.hasArgument(ATTACHMENT_PATHS)) {
```

Comment:
I suggest leaving the original `Intent.ACTION_SEND` flag when there is only one attachment in the list.